src/workloads/execution/PipelineUpdate.yml

```diff
@@ -0,0 +1,102 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
```

Comment:
Would it be possible to run these tests on different size documents? In my ideal world there'd be one collection of small documents (100-200 bytes) a collection of "medium" size documents (1-2kb) and collection of semi-large documents (100kb+). (The max document size is 16MB but I don't think we need to test documents that big). We could just run the same updates/operations on all three collections.

Reply:
Hmm  Yea we can do updates on specific size of documents. but it seems difficult for this "Loader" actor to load three collections and each collection contains only documents of same size (There is a 'CrudActor' that is able to run insertions with particular documents on a named collection, not sure if the actor can be used as a loader to load docs though.)
How about we have three different sizes of documents (small, medium, and semi-large) in **one** collection, and do different types of updates (something like "UpdateSmallSizeDoc", "UpdateMediumSizeDoc")?

Reply:
Sounds good!